workload/workloads/userProfile.go

```diff
@@ -171,44 +228,42 @@ func (w userProfile) findProfile(ctx context.Context, rctx workload.Runctx) erro
 }
 
 func (w userProfile) findRelatedProfiles(ctx context.Context, rctx workload.Runctx) error {
-	return nil
+	toFind := gofakeit.Paragraph(1, 1, rctx.Rand().Intn(12)+1, "\n") // one sentence to search
 
-	// toFind := gofakeit.Paragraph(1, 1, ctx.r.Intn(12)+1, "\n") // one sentence to search
-
-	// ctx.l.Sugar().Debugf("Searching for related profiles with string %s", toFind)
-	// params := make(map[string]interface{}, 1)
-	// params["email"] = toFind
-
-	// matchResult, err := scope.Search(
-	// 	"profile-statuses",
-	// 	search.NewMatchQuery(tofind),
-	// 	&gocb.SearchOptions{
-	// 		Limit:  10,
-	// 		Fields: []string{"description"},
-	// 	},
-	// )
-	// if err != nil {
-	// 	panic(err)
-	// }
-
-	// for matchResult.Next() {
-	// 	row := matchResult.Row()
-	// 	docID := row.ID
-	// 	score := row.Score
-
-	// 	var fields interface{}
-	// 	err := row.Fields(&fields)
-	// 	if err != nil {
-	// 		panic(err)
-	// 	}
-
-	// 	fmt.Printf("Document ID: %s, search score: %f, fields included in result: %v\n", docID, score, fields)
-	// }
-
-	// // always check for errors after iterating
-	// err = matchResult.Err()
-	// if err != nil {
-	// 	panic(err)
-	// }
+	rctx.Logger().Sugar().Debugf("Searching for related profiles with string %s", toFind)
```

Comment:
I think I didn't write enough of a description of what this operation is supposed to be.  The thought here was that you would find people who had related interests (e.g., similar company) or maybe were close to you (e.g., geo search).  It looks like because I copied/pasted the code for searching for someone, that's how the FTS search was implemented.

Glad to discuss, but I think I'd rather not put this change through until we have it doing what we want it to do.

Reply:
I'll hold off merging until we can discus